content/transfer-fee.md

```diff
@@ -0,0 +1,959 @@
+- `destination`: The destination account - in our case, the fee vault
+- `authority`: The mint's withdraw withheld tokens authority - in our case, the payer
+- `multiSigners`: Signing accounts if `owner` is a multisig
+- `sources`: Source accounts from which to withdraw withheld fees
+- `confirmOptions`: Options for confirming the transaction
+- `programId`: SPL Token program account - in our case `TOKEN_2022_PROGRAM_ID`
+
+Now, let's directly withdraw the fees from the destination account and check the resulting balances:
+
+```ts
+// DIRECTLY WITHDRAW
+await withdrawWithheldTokensFromAccounts(
+	connection,
+	payer,
+	mint,
+	feeVaultAccount,
+	payer.publicKey,
+	[],
+	[destinationAccount],
+	undefined,
+	TOKEN_2022_PROGRAM_ID
+);
+
+const withheldAccountAfterWithdraw = await getAccount(
+	connection,
+	destinationAccount,
+	undefined,
+	TOKEN_2022_PROGRAM_ID
+)
+
+const withheldAmountAfterWithdraw = getTransferFeeAmount(withheldAccountAfterWithdraw);
+
+const feeVaultAfterWithdraw = await getAccount(
+	connection,
+	feeVaultAccount,
+	undefined,
+	TOKEN_2022_PROGRAM_ID
+)
+
+console.log(`Withheld amount after withdraw: ${withheldAmountAfterWithdraw?.withheldAmount}`);
+console.log(`Fee vault balance after withdraw: ${feeVaultAfterWithdraw.amount}\n`);
+```
+
+Go ahead and run the script:
+```bash
+npm run start
+```
+
+You should get the following:
+```bash
+Withheld amount after withdraw: 0
+Fee vault balance after withdraw: 5000
+```
+
+Note: the `withdrawWithheldTokensFromAccounts` can also be used to collect all fees from all token accounts, if you fetch them all first. Something like the following would work:
+
+```ts
+const accounts = await connection.getProgramAccounts(
+  TOKEN_2022_PROGRAM_ID,
+  {
+    commitment: 'finalized',
+    filters: [
+      {
+        memcmp: {
+          offset: 0,
+          bytes: mint.toString(),
+        },
+      },
+    ],
+  }
+)
+
+const accountsToWithdrawFrom = []
+for (const accountInfo of accounts) {
+  const unpackedAccount = unpackAccount(
+    accountInfo.pubkey,
+    accountInfo.account,
+    TOKEN_2022_PROGRAM_ID
+  )
+
+  const transferFeeAmount = getTransferFeeAmount(unpackedAccount)
+  if (
+    transferFeeAmount != null &&
+    transferFeeAmount.withheldAmount > BigInt(0)
+  ) {
+    accountsToWithdrawFrom.push(accountInfo.pubkey)
+  }
+}
+
+await withdrawWithheldTokensFromAccounts(
+	connection,
+	payer,
+	mint,
+	feeVaultAccount,
+	payer.publicKey,
+	[],
+	accountsToWithdrawFrom,
+	undefined,
+	TOKEN_2022_PROGRAM_ID
+);
+```
+
+### 7.2 Harvest and then withdraw
+
+Let's look at the second option to retrieving the withheld fees: "harvesting". The difference here is that instead of withdrawing the fees directly, we "harvest" them back to the mint itself using `harvestWithheldTokensToMint`. This is a permissionless function, meaning anyone can call it. This is useful if you use something like [clockwork](https://www.clockwork.xyz/) to "crank" these harvesting functions. 
```

Comment:
```suggestion
Let's look at the second option to retrieving the withheld fees: "harvesting". The difference here is that instead of withdrawing the fees directly, we "harvest" them back to the mint itself using `harvestWithheldTokensToMint`. This is a permissionless function, meaning anyone can call it. This is useful if you use something like [clockwork](https://www.clockwork.xyz/) to automate these harvesting functions. 
```